src/client.py

```diff
@@ -1,29 +1,32 @@
 """Set up of our client."""
-
+# encoding: utf-8
 import socket
+import sys
 
 
 def main():
     """CONTROLER."""
-    import sys
     client(sys.argv[1])
 
 
 def client(message):
     """Connect client to server, send and receive message."""
-    message = message + "\r\n"
-    info = socket.getaddrinfo('127.0.0.1', 5011)
+    message = message + "\r\n\r\n"
+    if sys.version_info[0] == 2:
+        message = message.decode("utf8")
+    message = message.encode("utf8")
+    info = socket.getaddrinfo('127.0.0.1', 5000)
     stream_info = [i for i in info if i[1] == socket.SOCK_STREAM][0]
     client_ = socket.socket(*stream_info[:3])
     client_.connect((stream_info[-1]))
-    client_.sendall(message.encode('utf8'))
+    client_.sendall(message)
     buffer_length = 8
     response = ''
-    while response[-2:] != "\r\n":
+    while response[-4:] != "\r\n\r\n":
         response += client_.recv(buffer_length).decode('utf8')
```

Comment:
Similar to your server loop, it would be better to collect all the response parts, then decode them once to ensure you don't blow up any unicode characters.